problems.h

```diff
@@ -8,27 +9,172 @@ using namespace std;
 int reverseInt(int num)
```

Comment:
This is an interesting solution and it works, but I think you could've done it more efficiently without the to_string!